.ado/publish.yml

```diff
@@ -74,66 +74,3 @@ extends:
                persistCredentials: true # set to 'true' to leave the OAuth token in the Git config after the initial fetch
 
              - template: /.ado/templates/npm-publish.yml@self
-
-             # Set the git tag and push the version update back to Github
```

Comment:
We still need to keep this. This pushes changes back to the repo.

Reply:
Nx release will do this. I may have accidentally done so (and force push undone it) at least once already 😅

Reply:
You can check the (verbose) output of the dry run in my other PR to see when.

Reply:
Really? The documentation made no mention of it 😛